src/overlay/Peer.cpp

```diff
@@ -784,15 +784,16 @@ Peer::recvTransaction(StellarMessage const& msg)
     {
         // add it to our current set
         // and make sure it is valid
-        auto recvRes = mApp.getHerder().recvTransaction(transaction);
+        auto recvResStatus = mApp.getHerder().recvTransaction(transaction);
```

Comment:
Nitpick: The name reads as "receive result status" which is weird. How about just `recvStatus`?